recipe/meta.yaml

```diff
@@ -42,9 +40,7 @@ test:
     - python -m pip check 
   # downstreams:
     # Additional testing for downstream package jupyterlab
-    # 1. Skip s390x and arm64 because nodejs >=14,<15 currently isn't available.
-    # 2. Skip ppc64le: npm dependencies failed to install
-    #- jupyterlab  # [not (s390x or aarch64 or arm64 or ppc64le)]
+    #- jupyterlab
```

Comment:
we have set of test in upstream https://github.com/jupyterlab/jupyterlab_server/blob/v2.25.1/tests/, do not we want to execute them ?

Reply:
Unfortunately that would mean adding a good number of packages. I'll merge as is.